pum/core/dumper.py

```diff
@@ -40,6 +40,7 @@ def pg_restore(self, pg_restore_exe='pg_restore'):
 
         command = [
             pg_restore_exe, '-d',
-            'service={}'.format(self.pg_service), self.file]
+            'service={}'.format(self.pg_service), self.file,
+            '--exit-on-error']
```

Comment:
It may just make sense to add `--exit-on-error` before `self.file`, right after `-d` I would say.

Totally unrelated: I don't understand the use of `-d` here. Isn't `-d` used to specify the database to connect to? No database is specified after `-d` in this case!